lib/publish.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const execa = require('execa');
+const listrInput = require('listr-input');
+const Observable = require('any-observable');
+
+const npmPublish = opts => {
+	const args = ['publish'];
+
+	if (opts.tag) {
+		args.push('--tag', opts.tag);
+	}
+
+	if (opts.otp) {
+		args.push('--otp', opts.otp);
+	}
+
+	return execa('npm', args);
+};
+
+module.exports = tag => Observable.fromPromise(npmPublish({tag}))
+	.catch(err => {
+		if (err.stderr.indexOf('You must provide a one-time pass') !== -1) {
+			return listrInput('Enter OTP', {
```

Comment:
Is there a way to colorize `Enter OTP` here? I think it might be worth trying to make it stand out a little bit. I tried wrapping it in `chalk.yellow()` but that didn't change anything. For reference here's how the prompt looks on my machine vs how it looks when just doing `npm publish`:

<img width="267" alt="screen shot 2017-11-09 at 2 06 16 pm" src="https://user-images.githubusercontent.com/542108/32631141-8c12abf0-c55c-11e7-8e84-b0a3a69dcbc6.png">
<img width="227" alt="screen shot 2017-11-09 at 2 39 57 pm" src="https://user-images.githubusercontent.com/542108/32631142-8c405384-c55c-11e7-8ad8-aca1eb7221f4.png">

Reply:
No currently that's not possible. ANSI escape codes are stripped out of it. I hope that I am able to support that in the future.